conf/modules/raredisease.config

```diff
@@ -0,0 +1,148 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Any reason you didn't keep this general one as modules.config?

Reply:
Only modules we call directly from raredisease.nf are configured in this file and that's why we didn't call it modules.config.